### DISCUSSION TITLE
spawn latexmk ENOENT

### DISCUSSION THREAD

**Discussion by reserveword** [James-Yu/LaTeX-Workshop#3285] [Q&A]
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes/No

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes/No

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes/No

## Describe the bug [Required]

It 

### To Reproduce

3. See error

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: [e.g. Windows 10]
 - VS Code version: [e.g. 1.41.0]
 - Extension version: [e.g. 8.5.0]
 - TeX distribution version: [e.g. TeX Live 2019/MiKTeX 2.9.7250]

### Are you using VSCodium?

Yes/No

### Are you using the Snap or Flatpack versions of VS Code?

Yes/No

### Are you using LaTeX Workshop with VS Code Remote?

Yes/No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

**Comment by James-Yu** [comment]
https://github.com/James-Yu/LaTeX-Workshop/issues?q=is%3Aissue+spawn+latexmk+ENOENT

**Comment by reserveword** [comment]
sorry, it works normally after rebooting my computer
